hack/install-kube-apiserver.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+echo Installing kube-apiserver
+
+sudo mkdir -p /usr/local/kubebuilder/bin
+
+sudo curl -L https://dl.k8s.io/v1.16.4/kubernetes-server-linux-amd64.tar.gz -o /usr/local/kubebuilder/bin/kubernetes-server-linux-amd64.tar.gz
```

Comment:
This is k8s 1.16 ?

Reply:
I'll guess so. I suggest installing etcd and apiserver using https://github.com/kubernetes-sigs/controller-runtime/tree/master/tools/setup-envtest in a follow-up MR - that can also upgrade to a supported K8s version.

Reply:
I'm not sure. I ported what the previous automation had. I think we can do the upgrade later as @erikgb suggested

Reply:
agree with the team that we should upgrade the k8s version... but aligned with iblancasa that we can do that on the next PR